apps/main-chain/src/handlers/colonies/colonyAdded.ts

```diff
@@ -50,8 +50,9 @@ export default async (event: ContractEvent): Promise<void> => {
   /*
    * Add it to the Set
    */
+  // @NOTE seems to not be working, is it borken on master too?
```

Comment:
Seemed to work okay for me!

Before creating a new colony (tracked colonies 2)

![Screenshot 2024-12-11 at 17 07 46](https://github.com/user-attachments/assets/568427c1-5c9f-46e2-895e-e1abf0eed255)
 
After (tracked colonies 3!)

![Screenshot 2024-12-11 at 17 09 33](https://github.com/user-attachments/assets/4e7a4e13-b147-4491-9063-296fd3c51018)